specs/data_structures.md

```diff
@@ -320,31 +286,133 @@ A proof into an SMT is structured as:
 
 # Erasure Coding
 
+In order to enable trust-minimized light clients (i.e. light clients that do not rely on an honest majority of validating state assumption), it is critical that light clients can determine whether the data in each block is _available_ or not, without downloading the whole block itself. The technique used here was formally described in the paper [Fraud and Data Availability Proofs: Maximising Light Client Security and Scaling Blockchains with Dishonest Majorities](https://arxiv.org/abs/1809.09044).
+
+The remainder of the subsections below specify the [2D Reed-Solomon erasure coding scheme](#2d-reed-solomon-encoding-scheme) used, along with the format of [shares](#share) and how [available data](#available-data) is arranged into shares.
+
+## Reed-Solomon Erasure Coding
+
+TODO specify specifically
+
+## 2D Reed-Solomon Encoding Scheme
+
+![fig: RS2D encoding.](figures/rs2d.svg)
```

Comment:
That file seems missing or has another name.

Reply:
Yeah, it's one I'm working on and forgot to remove this line before pushing the commit!